addons/mail/static/src/core/common/chat_window.xml

```diff
@@ -58,7 +58,7 @@
         <div t-if="!props.chatWindow.folded" class="o-mail-ChatWindow-content bg-view d-flex flex-column h-100 overflow-auto position-relative" t-ref="content">
             <button t-if="threadActions.activeAction" class="o-mail-ChatWindow-closePanel btn position-absolute top-0 end-0 btn opacity-75 opacity-100-hover" title="Close panel" t-on-click.stop="() => threadActions.activeAction.close()"><i class="fa fa-close"/></button>
             <t t-if="thread" name="thread content">
-                <t t-if="threadActions.activeAction?.componentCondition" t-component="threadActions.activeAction.component" thread="thread" t-props="threadActions.activeAction.componentProps"/>
+                <t t-if="threadActions.activeAction?.componentCondition" t-component="threadActions.activeAction.component" thread="thread" t-props="threadActions.activeAction.componentProps?.(threadActions.activeAction)"/>
```

Comment:
As it's still not in the mergebot queue.

Passing action as param here is redundant.

The call should be done with param in `componentProps: action.componentProps,` line, so that getter is actually a getter and not a function to call everywhere

Reply:
This was the intended behavior, similar to other getters in this part of the code.